### PR TITLE
[FR] update `idempotence`

### PR DESCRIPTION
**Pull request by guillaumebernard84**
### Describe your changes
[FR] update `idempotence` following changes in English

### Related issue number or link (ex: `resolves #issue-number`)
N/A

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by Krast76 — APPROVED** [review]
LGTM 🚀

File: content/fr/idempotence.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ tags: ["propriété", "", ""]
 
 En mathématiques ou en informatique, l'idempotence décrit une opération qui aboutit toujours au même résultat,
 peu importe combien de fois vous l'exécutez.
-Si les paramètres sont identiques, une opération idempotente n'affectera pas l'application qu'elle appelle.
+Si les paramètres sont identiques, exécuter une opération idempotente plusieurs fois n'aura pas d'effet supplémentaire.
```
Comment (sestegra) [inline content/fr/idempotence.md:10]: Je propose une version légèrement différente
"n'aura aucun effet supplémentaire"

**Review by sestegra — CHANGES_REQUESTED** [review]

File: content/fr/idempotence.md (hunk 1 above)
Comment (guillaumebernard84) [reply content/fr/idempotence.md]: Tu as raison, ça me parait mieux. C'est corrigé

**Review by sestegra — APPROVED** [review]
LGTM 🚀

**Pull request merged**